boards/px4/fmu-v3/visionTargetEst.px4board

```diff
@@ -0,0 +1,7 @@
+CONFIG_MAVLINK_DIALECT="development"
+CONFIG_MODULES_VISION_TARGET_ESTIMATOR=y
+CONFIG_MODULES_LANDING_TARGET_ESTIMATOR=n
+CONFIG_MODULES_ROVER_POS_CONTROL=n
```

Comment:
This is kind of specific to your own project. You can simply keep that locally or just change it in menuconfig before building the firmware

Reply:
Due to the stack size limitations, I've had to disable certain modules to be able to build the frame. This custom frame is here to make sure that future implementations do not break the vision-target-estimator.

Note that eventually the goal is to keep the common Mavlink dialect (once the new Mavlink messages (target_relative and target_absolute) are moved from dev to common).